Src/FluentAssertions/Data/DataEquivalencyAssertionOptions.cs

```diff
@@ -0,0 +1,282 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq.Expressions;
+using System.Reflection;
+
+using FluentAssertions.Equivalency;
+
+namespace FluentAssertions.Data
+{
+    public class DataEquivalencyAssertionOptions<T> : EquivalencyAssertionOptions<T>
```

Comment:
Isn't better to extract an interface from this so only the members relevant for consumers are visible and not the properties such as `AllowMismatchedTypes` and such?

Reply:
I can get on board with that :-)